scripts/katana_version/__main__.py

```diff
@@ -723,7 +723,7 @@ def check_clean(args, config):
 
     is_dirty = git.is_dirty(config.open) or (config.has_enterprise and git.is_dirty(config.enterprise))
     if not args.clean and is_dirty:
-        raise StateError("Action only supported in clean repositories. (Stash your changes.)")
+        raise StateError("Action only supported in clean repositories. (Stash your changes or consider the --pretend-clean override option)")
```

Comment:
I intensionally didn't include this because in normal usage you should NOT use pretend. It can produce incorrect version change sets.

I would rather not have this included in this message.